node/module_test.ts

```diff
@@ -108,6 +108,9 @@ Deno.test("native modules are extensible", () => {
   const randomKey = "random-key";
   const randomValue = "random-value";
   for (const name of SUPPORTED_NODE_MODULES) {
+    if (name == "util/types") {
+      continue;
+    }
```

Comment:
reason: https://github.com/denoland/deno_std/pull/1791#issuecomment-1007386504

Reply:
How about doing `export default { ...types }` in `node/util/types.js` to avoid this workaround?

Reply:
https://github.com/denoland/deno_std/runs/4748029789?check_suite_focus=true
I've given it some thought, but I don't know how to solve it...

```zsh
-   {
+   Module {
      isAnyArrayBuffer: [Function: isAnyArrayBuffer],
```

Reply:
Ah, then, how about importing `./util/types.ts` instead of `./internal/util/types.ts` in `node/util.ts`? That should make those two objects the same thing, I guess

Reply:
Fixed :+1: https://github.com/denoland/deno_std/pull/1791/commits/44859270bb6b75a41dcaa2c10dfef78c1fb366b9